tiledbimg/converters/ome_zarr.py

```diff
@@ -1,24 +1,134 @@
+import glob
+import os
+import pickle
+from dataclasses import dataclass
+from typing import Any, Dict, Sequence
+
 import numpy as np
+import tiledb
+
+# import xmltodict
 import zarr
+from numcodecs import Blosc
+from ome_zarr.reader import Node, Reader, ZarrLocation
+from ome_zarr.writer import write_multiscale
+
+from .base import ImageConverter, ImageReader, ImageWriter
+
+
+@dataclass(frozen=True)
+class Level:
+    node: Node
+
+
+class OMEZarrWriter(ImageWriter):
+    def __init__(self, input_path: str, output_path: str):
+        self.input_path = input_path
+        self.output_path = output_path
+        # OME ZARR GROUP
+        self.image_group = zarr.group(
+            store=zarr.storage.DirectoryStore(path=self.output_path), overwrite=True
+        )
+        self.tldb_image = tiledb.Group(input_path, "r")
+        # levels is a list of TLDB ARRAYS
+        self._levels = sorted([resolution.uri for resolution in self.tldb_image])
+        self.level_arrays = [lev_uri for lev_uri in self._levels]
+
+    def uri(self, level: int) -> str:
+        return os.path.join(self.input_path, f"{level}")
+
+    @property
+    def level_count(self) -> int:
+        return len(self.tldb_image)
+
+    def level_image(self, level: int) -> np.ndarray:
+        with tiledb.open(self._levels[level]) as L:
+            temp_array = L[:].swapaxes(0, 2)
```

Comment:
After https://github.com/TileDB-Inc/TileDB-BioImaging/pull/26 data are stored in CYX order, no need for swapaxes here.